### ISSUE TITLE
Low accuracy on Kinetics (2D resnet is better than 3D resnet)

### ISSUE THREAD

**Issue by gurkirt** [kenshohara/3D-ResNets-PyTorch#88]
I trained a 2D resnet101 on kinetics dataset, and it has much better accuracy than your 3D resent. I used https://github.com/gurkirt/2D-kinectics to train 2D resnet. Of course, pretrained on imagenet. 

I got the top1 accuracy of 70.3 on kinetics-400.  
Do you think it is because of pretraining? 
Have you tried  2D networks with your training procedure?

**Comment by bityangke** [comment]
In the Non-local Neural Network paper, the top 1 acc of 2D-ResNet50 baseline is 71.8. 
@gurkirt @kenshohara 
I believe the performance gap does not come from the ImageNet pretraining as @kenshohara  said.

**Comment by kkjh0723** [comment]
@kenshohara @gurkirt @bityangke 
Did anyone find the reason for the difference?
